### ISSUE TITLE
Gotcha: Determining p value

### ISSUE THREAD

**Issue by motiwari** [motiwari/BanditPAM-python#20]
Introducing the exact re-computation when T_samples >= n introduced a small problem. The "true" medoid was sometimes dropped; this was happening because the true medoid (e.g. 595's) lcb was higher than the EXACTLY computed point's lcb = ucb = estimate. We needed to complete more estimates to put the estimate for 595 below the exact point, but due to random noise it was initially placed above the exact point and then wasn't sampled any further.

We fixed this problem by widening the confidence interval, i.e. lowering p, so the probability that 595's true value was actually in its confidence interval was much higher -- also its confidence interval was much wider so it didn't get "ruled out" when computing points exactly.

In summary, the problem happened because 595's true distance was less than some other point's X, but at some point, it's confidence interval was higher than X, and X was computed exactly first, lowering the ucb to below 595's lcb
